src/models/Article.php

```diff
@@ -115,10 +122,17 @@ public function attributeLabels()
      */
     public function ngRestAttributeTypes()
     {
+        if (Yii::$app->controller->module->id == 'newsadmin') {
```

Comment:
@Webkadabra i can see the idea behing this, but we should not use fix confiugration inside a model. So maybe we should allow people to override ngrest configurations trough module configuration in general. So we won't have such problems. I was describe such a concept already a long time ago: https://github.com/luyadev/luya-module-admin/blob/master/src/base/Module.php#L63-L95

Reply:
So in order to merge the PR, we should just swtich back to the implementation used before.